### PR TITLE
fix symbol priority problem

### PR DESCRIPTION
**Pull request by yu-zhy-neusoft**
Add parentheses to the right expression of "& &"

Test: test ok
Signed-off-by: yuzhengyang <yu.zhy@neusoft.com>

### CODE REVIEW

**Comment by Shao-Feng** [comment]
Please add the field "Tracked-On" for commit.

File: os/android/iahwc2.cpp [hunk 1]
```diff
@@ -682,7 +682,7 @@ HWC2::Error IAHWC2::HwcDisplay::PresentDisplay(int32_t *retire_fence) {
     layers.emplace_back(l.second->GetLayer());
   }
 
-  if (layers.empty() && display_->Type() != DisplayType::kLogical)
+  if (layers.empty() && (display_->Type() != DisplayType::kLogical))
```
Comment (YuanjunHuang) [inline os/android/iahwc2.cpp]: existing expression is correct and clear enough. no need to submit a dedicated pull request to change it.
but of coz you are welcome to refine the code.
